crispy_forms/helper.py

```diff
@@ -337,10 +337,12 @@ def get_attributes(self, template_pack=TEMPLATE_PACK):  # noqa: C901
 
         if template_pack == "bootstrap4":
```

Comment:
What do you think about adding this whole section to a new function? Would it make helper simpler, and this piece easier to test?

Reply:
I am not sure that it is necessary: I extended the test checking the column are well transmitted for BS4 with 7ac2f7f. Test with BS3 have already been written [here](https://github.com/django-crispy-forms/django-crispy-forms/pull/1008/files#diff-46a067d22cfed9b9ab34736364f083c5R788-R816).

Reply:
ok 👍